dist/s_all

```diff
@@ -45,33 +69,49 @@ force=
 errmode=0
 errfound=0
 
-while :
+help() {
+    cat << _END
+Run all presubmit tests.
+Usage: $0 [... options]
+  -E                 Return an error code on failure.
+  -f                 Force versions to be updated.
+  -F                 Run fast.
+  --no-interactive   Disable interactitve terminal status.
+  -h, --help         Display this help.
+_END
+    exit $1
+}
+
+while [[ "$#" -gt 0 ]]
     do case "$1" in
-    -E)    # Return an error code on failure
+    -E)                # Return an error code on failure
         errmode=1
         shift;;
-    -f)    # Force versions to be updated
+    -f)                # Force versions to be updated
         force="-f"
         shift;;
-    -F)    # Run fast.
+    -F)                # Run fast.
         echo "dist/s_all running in fast mode..."
         fast="-F"
         shift;;
+    -h | --help) help 0;;
+    --no-interactive)   # Force disable terminal status report
+        ISTTY="" CLRIFTTY=""
+        shift;;
     *)
-        break;;
+        echo "Unknown option '$1'. Use '-h' for help."
+        exit 1;;
     esac
 done
 
-echo "Updating files that include the package version" &&
-    bash ./s_version $force
-
 errchk()
 {
-    if ! `test -s $2`; then
+    if [[ ! -s $2 ]]; then
```

Comment:
Can we assign `$2` to a variable, it is hard to know what it corresponds to.

Reply:
Done